docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -255,12 +253,29 @@ the game ends in a draw:
       X | O | X
 
 
-XO Transaction Family Specification
-===================================
+Step 7. Delete the Game
+-----------------------
+
+Either user can use the ``xo delete`` command to delete their local XO data.
+This includes all games, the saved URL, and the username.
```

Comment:
doesn't that delete the game from the blockchain's state? does XO have local data?

Reply:
Alas, I can't answer this question (I don't know the internals of the XO transaction family).  Can anyone else provide the answer to Dan's question?

Reply:
I double checked and yes `delete` is a transaction. that method will remove the game from global state. The global state is defined as `<name>,<board>,<game-state>,<player-key-1>,<player-key-2>` 
You'll want to completely strike the line "This includes all games, ... username."
Incidentally the family spec does not list this delete operation. I'll open a jira ticket for that.